### ISSUE TITLE
Unclear how to use templating feature

### ISSUE THREAD

**Issue by kwatsen** [moinwiki/moin#870]
The `Templates and Meta Data` page in document describes how templates must contain the tag `template`, but it doesn't explain how to use a template.  

Specifically, after creating a top-level "MyTemplate" page and setting its tag to "template", I then tried to create another page hoping that the "Please select the contenttype of the new default item." page might expand to offer "Template" as an option.  But this didn't happen.

In the end, I'm unsure how the templating feature is suppose to work.

**Comment by RogerHaase** [comment]
Cannot reproduce.

Did you read "Only templates having the same content type and namespace as the new item will be shown as choices." within https://moin-20.readthedocs.io/en/latest/user/templates_metadata.html?highlight=template#templates-and-meta-data

So if you create a template with moin-wiki contenttype in the default namespace, it will not be offered as a choice when you create an item with a markdown contenttype, nor if you create a moin-wiki contenttype in the "users" namespace. 

The template choice view will appear after the "select contenttype" view:

    Create new item?

    You can either create the item from scratch or select a template.
    Available template items

        TemplateSample

**Comment by kwatsen** [comment]
Okay, I tried again and it workes.  My guess is that it would've worked last time too, had I actually clicked the "MoinMoin" link on the page before.

I was expecting the option to select a template to appear on the previous page, not after selecting a content type...

This is what I meant about the docs being improved, a few screenshots and/or some additional text would've helped immensely.   

Still not clear to me if there is a preferred location for where templates should be located.  Perhaps a recommendation to keep them close to the pages they are used to create?

Since this is just a documentation issue, perhaps assign to me?
